python/lsst/cp/pipe/cpDark.py

```diff
@@ -114,19 +124,38 @@ def run(self, inputExp):
         # exposure, if possible.  Otherwise, use the cameraGeom value.
         gains = self._get_gains(inputExp)
 
+        # Is this gainContext load bearing?
         with gainContext(inputExp, inputExp.getVariance(), apply=True, gains=gains):
             # Scale the variance to match the image plane.  A similar
             # scaling happens during flat-field correction for science
             # images.
             self.log.debug("Median image and variance: %f %f",
                            np.median(inputExp.image.array), np.median(inputExp.variance.array))
+            crImage = inputExp.clone()
+            # Interpolate the crImage, so the CR code can ignore
+            # defects (which will now be interpolated).
```

Comment:
Are you putting this here so that we can use the new interpolation code if we turn it on?  Otherwise interpolation will be done by the CR task below if you supply the `defects`

Reply:
I think I confused myself over the course of this ticket's life and general confusion last week.  I've dropped the passing of constructed defects to the CR code, as this interpolation is more flexible (we can switch to the new interpolation if desired).  I also see that the code from last week wasn't using the `maskListToInterpolate` config option, so this version appears more robust.

Reply:
Good point, the other less flexible method would only do BAD.  This is configurable, so better.